plugins/database/mssql/mssql_test.go

```diff
@@ -411,6 +458,30 @@ func assertCredsDoNotExist(t testing.TB, connURL, username, password string) {
 	}
 }
 
+func assertContainedDBCredsExist(t testing.TB, connURL, username string) {
+	t.Helper()
+	err := testContainedDBCredsExist(connURL, username)
+	if err == nil {
+		t.Fatalf("Able to drop user when it shouldn't")
+	}
```

Comment:
I think we may want to do some checking on the actual content of this error, too, since it could be any number of things. (For instance right now it's actually `sql: expected 0 arguments, got 1`.)

Reply:
f782d30 Adds a check of the error message, this also made me change the function name since we are trying to ensure that there is an error - `assertContainedDBCredsDoNotExist`